src/Face.cuh

```diff
@@ -44,12 +45,26 @@ public:
     __device__ ~Face();
 
 
-    /// An identifier of the face of a `Polyhedron`
-    const int id;
+    /**
+     * Attaches given node to the face, if the node lays on it, nothing happens otherwise
```

Comment:
I think that you did not read it properly, but I will change the word order

Reply:
@tanya-kta, Ooops, sorry, you are right. I'm sorry, the sentence is just ok, but I think, you have to clarify what "node lays on it" means. At the moment it can be understood as if the node is attached to the face, then attach it again, do nothing otherwise.